app/models/patient.rb

```diff
@@ -235,12 +245,30 @@ def latest_transfer
       .where(isolation: true)
       .where('symptom_onset <= ?', 10.days.ago)
       .where(latest_fever_or_fever_reducer_at: nil)
+      .where(extended_isolation: nil)
       .or(
         where(monitoring: true)
         .where(purged: false)
         .where(isolation: true)
         .where('symptom_onset <= ?', 10.days.ago)
         .where('latest_fever_or_fever_reducer_at < ?', 24.hours.ago)
+        .where(extended_isolation: nil)
+      )
+      .or(
```

Comment:
This seems like a lot of added complexity (too many `or`s). Perhaps instead try a `where.not`? `.where.not('extended_isolation >= ?', Date.today)` or something like that (not tested).

Reply:
I wanted to do that originally and I tried it but it wasn't working for me

Reply:
`where.not` [won't consider ](https://thoughtbot.com/blog/activerecord-s-where-not-and-nil)`nil` values, which is probably why it wasn't working for you. Take a look at the line here I used to avoid this exact scenario in the optimal version of `reminder_eligible`: https://github.com/SaraAlert/SaraAlert/blob/master/app/models/patient.rb#L100

Reply:
ah ok, thanks @kylietmo , I'll give that a try!

Reply:
resolved here: https://github.com/SaraAlert/SaraAlert/pull/336/commits/a8b4e62a8ab0cc8db0c6d9aa5c9ebd9a9411646b